service/src/main/webapp/assets/css/login.css

```diff
@@ -31,7 +31,7 @@ body {
     z-index: 2;
 }
 .form-signin input[type="text"] {
-    margin-bottom: -1px;
+    margin-bottom: 10px;
```

Comment:
This change messed up the spacing between boxes. @CatalystOfNostalgia would you mind looking at this and correcting it?